packages/terra-horizontal-divider/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "terra-horizontal-divider",
+  "main": "lib/HorizontalDivider.js",
+  "version": "0.0.0",
+  "description": "The terra-horizontal-divider component is a component that is used to separate content on a layout.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-horizontal-divider",
+    "HorizontalDivider",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^1.0.0"
```

Comment:
should be `"terra-base": "^2.6.1"`

Reply:
Updated https://github.com/cerner/terra-core/pull/962/commits/b6a950329f831a533a9f38d54427d9c17a5e17e6#diff-3fe4bec3374f1238c4e258d3c3e199feR25